kobotoolbox/kpi
# Fix stuck legacy exports issue introduced in `2.024.25`

## Description

Exports no longer stay "Pending ..." 

## Notes

MongoDB was maxed-out at 125 ms instead of 2100 seconds in async tasks or 120 seconds in sync tasks.
Two classes `MongoHelper` were co-existing in the code base. The one from KoboCAT was removed in favor the KPI one. 


## Testing

- Checkout `2.024.25b`
- Add several submissions to a project
- Try to export with Legacy XLS or Legacy CSV
- See that Celery crashes with error `pymongo.errors.ExecutionTimeout`
- Checkout this PR branch
- Retry 
- Everything should be ok
